### ISSUE TITLE
IDISP016 should not warn in loop

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#255]
```cs
            foreach (var conversion in this.AllConversions)
            {
                conversion.Dispose();
            }
```
